src/Rings/groebner/general.jl

```diff
@@ -55,6 +55,7 @@ The keyword `algorithm` can be set to
 - `:fglm` (implementation of the FGLM algorithm in *Singular*),
 - `:hc` (implementation of Buchberger's algorithm in *Singular* trying to first compute the highest corner modulo some prime), and
 - `:hilbert` (implementation of a Hilbert driven Gröbner basis computation in *Singular*).
+- `:g4ti2` (implementation to compute Gröbner basis for lattice ideals in *4ti2*)
```

Comment:
@wdecker @jankoboehm Any problem with the symbol `:g4ti2` for this groebner algorithm option?

Reply:
> @wdecker @jankoboehm Any problem with the symbol `:g4ti2` for this groebner algorithm option?

Fine with me.

Reply:
> Fine with me.

After thinking about it again: We should not use a particular dependancy such as 4ti2 here because this dependancy might change. How about using `:markov` or `markov_basis`?

Reply:
I agree